apps/governance/src/routes/proposals/components/proposal/proposal.tsx

```diff
@@ -36,13 +37,16 @@ export interface ProposalProps {
   assetData?: AssetQuery | null;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   restData: any;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  originalMarketProposalRestData?: any;
```

Comment:
**nitpick (non-blocking)**: I feel like somewhere we will have the type for a market proposal, and that if we did type it properly we might catch errors in the future (say `proposal.terms` changes). But the way the data is handled is careful enough to cope and the JSONDiff component would be fine so I think it's ok *this time*